components/zeus_ui/events/ui_spawn.sqf

```diff
@@ -86,10 +86,11 @@ case "ui_spawn": {
 					_suppressiveAI = _presetNamespace getVariable [MACRO_VARNAME_PRESET_SAI, []];
 				};
 
+				_dir = getDir curatorCamera;
 
 				// Tell the server to spawn the group
 				f_fnc_server_spawnGroup = compile preprocessFileLineNumbers "components\zeus_ui\fn_server_spawnGroup.sqf";
```

Comment:
Surprised this is still here. I removed the same thing from F3_CA a while back since it was originally a debug line, perhaps use the opportunity to clean this up aswell?

Reply:
Took it out, thanks for the review